ext/autocomplete/main.php

```diff
@@ -13,21 +13,28 @@ public function onPageRequest(PageRequestEvent $event) {
 			if(!isset($_GET["s"])) return;
 
 			//$limit = 0;
+			$cache_key = "autocomplete-" . strtolower($_GET["s"]);
 			$limitSQL = "";
 			$SQLarr = array("search"=>$_GET["s"]."%");
 			if(isset($_GET["limit"]) && $_GET["limit"] !== 0){
 				$limitSQL = "LIMIT :limit";
 				$SQLarr['limit'] = $_GET["limit"];
+				$cache_key .= "-" . $_GET["limit"];
 			}
 
-			$res = $database->get_pairs("
-				SELECT tag, count
-				FROM tags
-				WHERE tag LIKE :search
-				AND count > 0
-				ORDER BY count DESC
-				$limitSQL", $SQLarr
-			);
+			$res = null;
+			$database->cache->get($cache_key);
```

Comment:
`$res = $database->cache->get($cache_key);`

(My bad I think? Was trying various combinations of caching rules >_>)